app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,24 @@
+class PostsController < ApplicationController
+  def show
+    @post = Post.find(params[:id])
+  end
+
+  def new
+  end
+
+  def create
+    @post = Post.new(post_params)
```

Comment:
Please use `Post.create....` and then `@post.persisted?`

Reply:
done